### PR TITLE
Docs: fix some Sphinx warnings

### PR DESCRIPTION
**Pull request by jaredgrubb**

### CODE REVIEW

File: master/docs/manual/cfg-dbconfig.rst [hunk 1]
```diff
@@ -11,7 +11,9 @@ DbConfig can get and store any jsonable object to the db for use by other master
 The design is voluntary simplistic, the focus is on the easy use rather than efficiency.
 A separate db connection is created each time get() or set() is called.
 
-Example ::
+Example:
```
Comment (sa2ajj) [inline master/docs/manual/cfg-dbconfig.rst]: I'm really curious about these... :)

File: master/docs/manual/cfg-wwwhooks.rst [hunk 2]
```diff
@@ -77,7 +79,9 @@ The parameters are:
 
 :guilabel:`Secret`
     Any value.
-    If you provide a non-empty value (recommended), make sure that your hook is configured to use it::
+    If you provide a non-empty value (recommended), make sure that your hook is configured to use it:
+
+.. code-block:: python
```
Comment (sa2ajj) [inline master/docs/manual/cfg-wwwhooks.rst]: Indentation is wrong here.

File: master/docs/manual/cfg-wwwhooks.rst [hunk 3]
```diff
@@ -123,7 +129,7 @@ BitBucket hook
 
 The BitBucket hook is as simple as GitHub one and it also takes no options.
 
-::
+.. code-block:: python
```
Comment (sa2ajj) [inline master/docs/manual/cfg-wwwhooks.rst]: "Explicit is better than implicit." :)

**Comment by sa2ajj** [comment]
Beside the indentation, LGTM

File: master/docs/manual/cfg-dbconfig.rst (hunk 1 above)
Comment (jaredgrubb) [inline master/docs/manual/cfg-dbconfig.rst]: Sphinx was complaining that the block that followed had no language assigned to. So, adding the block means the :: moves down to that markup.

File: master/docs/manual/cfg-wwwhooks.rst (hunk 2 above)
Comment (jaredgrubb) [inline master/docs/manual/cfg-wwwhooks.rst]: I'm not sure I follow?

Comment (sa2ajj) [inline master/docs/manual/cfg-wwwhooks.rst]: the `.. code-block:: python` along with the actual code should be indented on the same leve as "If you provide", your change brings the code example one indentation level less.

**Comment by jaredgrubb** [comment]
Alright, fixed that indent and fixed a few more warnings that snuck in since.

**Pull request merged**